### PR TITLE
Persist blobs info to BAR

### PR DESCRIPTION
**Pull request by JohnTortugo**
As talked with @jcagme, this code, for now, when publishing to an AzDO target feed will only save the assets information to BAR. The actual publishing will be performed through YAML tasks.

### CODE REVIEW

**Comment by JohnTortugo** [comment]
Tested here: https://dnceng.visualstudio.com/internal/_build/results?buildId=238046&view=results

File: src/Microsoft.DotNet.Build.Tasks.Feed/src/PublishArtifactsInManifest.cs [hunk 1]
```diff
@@ -311,14 +374,67 @@ private void SplitArtifactsInCategories(BuildModel buildModel)
...
-        
+
+        private async Task PublishBlobsToAzureStorageNugetFeedAsync(
+            List<BlobArtifactModel> blobsToPublish,
+            IMaestroApi client,
+            Maestro.Client.Models.Build buildInformation,
+            FeedConfig feedConfig)
+        {
+            BlobAssetsBasePath = BlobAssetsBasePath.TrimEnd(Path.DirectorySeparatorChar,
+                Path.AltDirectorySeparatorChar) + Path.DirectorySeparatorChar;
```
Comment (jcagme) [inline src/Microsoft.DotNet.Build.Tasks.Feed/src/PublishArtifactsInManifest.cs]: Nit: move params to independent lines

**Comment by jcagme** [comment]
>                 return "NetCore";

NETCORE

---
Refers to: src/Microsoft.DotNet.Build.Tasks.Feed/src/PublishArtifactsInManifest.cs:455 in 7e38384. [](commit_id = 7e38384f7e82aec1150d9ad8d26cdc651e011272, deletion_comment = False)

**Comment by jcagme** [comment]
>                 return "NetCore";

You could even create a struct/class with constant known categories/extensions

---
In reply to: [505519265](https://github.com/dotnet/arcade/pull/3156#issuecomment-505519265) [](ancestors = 505519265)

---
Refers to: src/Microsoft.DotNet.Build.Tasks.Feed/src/PublishArtifactsInManifest.cs:455 in 7e38384. [](commit_id = 7e38384f7e82aec1150d9ad8d26cdc651e011272, deletion_comment = False)

**Comment by JohnTortugo** [comment]
> You could even create a struct/class with constant known categories/extensions

I'm going to address that later as I'd have to test this all again. Also, I've plans to integrate these constants with the values used in the .proj file, maybe by using an ItemGroup in a .props file.

**Comment by jcagme** [comment]
:heavy_check_mark: 

---
In reply to: [505522838](https://github.com/dotnet/arcade/pull/3156#issuecomment-505522838) [](ancestors = 505522838)

**Pull request merged**